infra/base-images/base-builder/targets_list

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+for binary in $(find $OUT/ -executable -type f); do
```

Comment:
for an additional layer of safety I'd also use `-maxdepth 1`, e.g. https://github.com/google/oss-fuzz/blob/218f5dc3cf70bd697105f754418e98119d2b9a8d/infra/base-images/base-runner/coverage#L22

Reply:
hmm, this doesn't match our behaviour on ClusterFuzz. Perhaps we should relax this to match that?

Reply:
I seem to remember adding os.walk to support fuzz targets in a subdirectory, but can't remember exactly which project(s) it was for. 

Let's make things match CF for now.